stages/org.osbuild.users

```diff
@@ -125,6 +126,9 @@ def main(tree, options):
         else:
             useradd(tree, name, uid, gid, groups, description, home, shell, password, expiredate)
 
+        if password_changed_date is not None:
+            subprocess.run(["chroot", tree, "chage", "--lastday", str(password_changed_date), name], check=True)
```

Comment:
A tiny unit test would be nice, I attached what I have in mind here.
[0001-stages-users-add-small-unit-test-that-chage-is-calle.patch.txt](https://github.com/osbuild/osbuild/files/14833219/0001-stages-users-add-small-unit-test-that-chage-is-calle.patch.txt)

Reply:
Added!